api/v1/models.go

```diff
@@ -95,9 +95,16 @@ func featureFromDatabaseModel(dbFeatureVersion database.FeatureVersion, uncertif
 	}
 }
 
-func hasKernelPrefix(name string) bool {
+// isPackageBlacklistedInScan returns true if the given package name in the
+// namespace should be skipped in vulnerability matching scans.
+func isPackageBlacklistedInScan(namespace, pkgName string) bool {
```

Comment:
nit: let's change the name to more inclusive language. I think we have been using `blocklist`